rust/build.rs

```diff
@@ -11,7 +11,7 @@ fn generate_shardsc() {
 
   let shards_dir = var("SHARDS_DIR").unwrap_or_else(|_| "..".to_string());
   let shards_include_dir = format!("{}/include", shards_dir);
-  let src_extra_dir = format!("{}/src/extra", shards_dir);
+  let src_extra_dir = format!("{}/src/extra", shards_dir); 
```

Comment:
Looks like a parasite change: not sure why there should be a whitespace at the end.